CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# MobileCenter-SDK-iOS Change Log
+# mobile-center-sdk-ios Change Log
```

Comment:
You could use the proper SDK name "Mobile Center SDK for iOS" here instead of the repo name. Looks a bit nicer imo 🙂